os/kernel/binary_manager/binary_manager_data.c

```diff
@@ -191,6 +191,13 @@ void binary_manager_add_binlist(FAR struct tcb_s *tcb)
 {
 	int bin_idx;
 
+	DEBUGASSERT(tcb);
+
+	if (!tcb->group) {
```

Comment:
Good to add a one line comment, why this check is important here.

Reply:
Done